contrib/create-waku/package.json

```diff
@@ -1,5 +1,22 @@
 {
   "name": "create-waku",
   "version": "0.4.10",
-  "bin": "./cli.js"
+  "bin": "out.js",
+  "type": "commonjs",
+  "scripts": {
```

Comment:
Let's add this:

```suggestion
  "files": [
    "src",
    "dist"
  ],
  "scripts": {
```

Reply:
Should we also include ``template`` in this?

Reply:
Oh, good point. We need that too.